### ISSUE TITLE
Upgrade the extension to the latest TS support

### ISSUE THREAD

**Issue by egamma** [Microsoft/vscode-ts-tslint#7]
TS 2.9 has added some additional support to improve running tslint as an extension:
https://github.com/Microsoft/TypeScript/issues/15914
https://github.com/Microsoft/TypeScript/issues/15915

**Comment by RDeluxe** [comment]
Does this mean this plugin will be updated soon ? 
From what I understand, this plugin -based on the ts-language-service - is more future proof than the other version, right ?

**Comment by egamma** [comment]
>Does this mean this plugin will be updated soon ?

yes this is the plan. Notice that you can use the ts-language-service plugin independent of the extension using this approach  https://github.com/Microsoft/vscode-ts-tslint/blob/master/README.md#using-a-different-version-of-tslint-than-the-version-that-is-bundled-with-the-extension

>From what I understand, this plugin -based on the ts-language-service - is more future proof than the other version, right ?

yes and no, the differences are described here https://github.com/Microsoft/vscode-ts-tslint/blob/master/README.md#differences-between-vscode-tslint-vnext-and-vscode-tslint, but there are still some limitations https://github.com/angelozerr/tslint-language-service/issues/32.
